com.unity.render-pipelines.universal/Runtime/Passes/AdditionalLightsShadowCasterPass.cs

```diff
@@ -832,6 +849,25 @@ void Clear()
             m_AdditionalLightsShadowmapTexture = null;
         }
 
+        void EmptyAdditionalShadowmapAtlas(ref ScriptableRenderContext context)
```

Comment:
Can we also improve this function name as well?
Something like:
```suggestion
        void SetEmptyAdditionalShadowmapAtlas(ref ScriptableRenderContext context)
```

Reply:
I will add Execute as prefix, setup might be confusing with our existing setup functions

Reply:
Of course render would be more consistant, but might be confussing as there is not rendering :/. Naming is hard...

Reply:
I think `Set` is different enough from `Setup` prefix